types/amazon-connect-streams/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```

Comment:
All new package definitions need to enable `strictNullChecks` so please enable and fix any errors.

Reply:
Sorry! I had disabled it for the test script. Re-enabled and tweaked the test file.